cli/README.md

```diff
@@ -202,6 +203,49 @@ EXAMPLE
 _See code: [src/commands/sync.ts](https://github.com/mirego/accent/blob/v0.11.0/src/commands/sync.ts)_
 <!-- commandsstop -->
 
+# GitHub Actions
+
+In addition to syncing the translations manually, you can add a GitHub Actions workflow to your project in order to automate the process.
+
+## Example
+
+```
+name: Accent
+
+on:
+  schedule:
+    - cron: "0 4 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - run: npm install -g accent-cli
+      - run: accent sync --add-translations --merge-type=passive --order-by=key
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:
+          add-paths: |
+            *.json
+          commit-message: Update translations
+          committer: Accent <accent@mirego.com>
+          author: Accent <accent@mirego.com>
```

Comment:
I think we should suggest this:

```suggestion
          committer: github-actions[bot]@users.noreply.github.com
          author: github-actions[bot]@users.noreply.github.com
```

to explicitly state that the commit was made by a bot.